lib/simperium/bucket-store.js

```diff
@@ -67,7 +67,7 @@ BucketStore.prototype.get = function(id, callback) {
 	});
 };
 
-BucketStore.prototype.update = function(id, data, callback) {
+BucketStore.prototype.update = function(id, data, isIndexing, callback) {
```

Comment:
This method is called by the Simperium bucket object. I was sincing errors in the console that `callback` was not a function.